docs/source/api/plugin/subscription-callback.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: "API Reference: Subscription callback plugin"
+api_reference: true
+minVersion: 4.9.0
+---
+
+> This protocol is currently in [preview](/resources/product-launch-stages#preview). Breaking changes might be introduced during the preview period.
```

Comment:
Without prior reference, "this protocol" is a little confusing.

I would put this sentence after the next one ("This document...") To maintain the importance of it, I would bold it.

```suggestion
```

Reply:
Agreed, much better on the next line. I added a **⚠️ Note:** in front of it. [33bb766](https://github.com/apollographql/apollo-server/pull/7666/commits/33bb7669841ccc7861e22b2c13110d31b90873a0)